README.md

```diff
@@ -71,6 +71,15 @@ extraEnvs:
     value: "true"
 ```
 
+OpenSearch version 2.12 and above require a custom strong password to be provided in order to setup demo admin user. Without this password the cluster would not spin up, unless demo config install is disabled.
```

Comment:
This should be highlighted as ## Breaking Changes or something similar, or at the top of the readme or users will miss it easily.